### ISSUE TITLE
[SUGGESTION] Mods that could have compatibility with PrideGUI

### ISSUE THREAD

**Issue by Thinkseal** [Pridecraft-Studios/prideGUI#10]
## Checklist

- [x] Mod Menu
- [ ] No Chat Reports
- [ ] Fabric Loader/API
    - [ ] Tab Switching Buttons
- [ ] Roughly Enough Items
- [x] Enchancement
- [x] Inventory Profiles Next
- [x] Animal Feeding Trough
- [ ] Create
    - [ ] Fabric
    - [ ] Forge
- [x] Origins
    - [x] Fabric
    - [x] Forge
- [ ] Supplementaries
- [ ] Amethyst Imbuement
- [ ] Axiom
- [ ] Sidekick
## Source code links and screenshots

[Mod Menu](https://github.com/TerraformersMC/ModMenu)

<img width="748" alt="Screenshot 2024-11-05 at 12 05 45 PM" src="https://github.com/user-attachments/assets/4de65a2e-f49b-452e-90ff-890bf74abf01">
<img width="412" alt="Screenshot 2024-11-05 at 12 12 56 PM" src="https://github.com/user-attachments/assets/01134003-316c-4565-9ad6-c5efc9d94a2e">

[No Chat Reports](https://github.com/Aizistral-Studios/No-Chat-Reports)
<img width="1018" alt="Screenshot 2024-11-05 at 12 06 12 PM" src="https://github.com/user-attachments/assets/fd1042f7-e81f-4109-8008-24bf65b20c4c">

Fabric's tab switching buttons

<img width="72" alt="Screenshot 2024-11-05 at 12 07 22 PM" src="https://github.com/user-attachments/assets/0d6bc67d-0deb-48bf-a460-ad6b245b9986">

[Roughly Enough Items](https://github.com/shedaniel/RoughlyEnoughItems)

<img width="113" alt="Screenshot 2024-11-05 at 12 08 16 PM" src="https://github.com/user-attachments/assets/5143b8d2-dbd6-44e1-a819-6fac072fb357">
<img width="464" alt="Screenshot 2024-11-05 at 12 08 39 PM" src="https://github.com/user-attachments/assets/1ae0aee5-1b4c-4b1b-8ad8-6b80e5399e19">
<img width="621" alt="Screenshot 2024-11-05 at 12 08 57 PM" src="https://github.com/user-attachments/assets/ec03c952-e0c6-4b46-a32d-577b650f8200">

[Enchancement](https://github.com/MoriyaShiine/enchancement)

<img width="515" alt="Screenshot 2024-11-05 at 12 09 43 PM" src="https://github.com/user-attachments/assets/d3ce585c-319d-43a4-8f06-e1eed23966a8">

[Inventory Profiles Next](https://github.com/blackd/Inventory-Profiles)

<img width="204" alt="Screenshot 2024-11-05 at 12 12 24 PM" src="https://github.com/user-attachments/assets/3e73cca0-8404-49dc-80d5-cdb40879bd91">

[Animal Feeding Trough](https://github.com/Slexom/animal-feeding-trough)

<img width="528" alt="Screenshot 2024-11-05 at 12 20 08 PM" src="https://github.com/user-attachments/assets/460f58ea-af2f-4169-b841-d795245bb79b">

[Create](https://github.com/Creators-of-Create/Create) ([Create Fabric](https://github.com/Fabricators-of-Create/Create))

<img width="636" alt="Screenshot 2024-11-05 at 12 22 27 PM" src="https://github.com/user-attachments/assets/1b3069ba-4954-42c5-b073-0cd040f078be"> 

[Origins ](https://github.com/apace100/origins-forge) ([Origins Fabric](https://github.com/apace100/origins-fabric))

<img width="750" alt="Screenshot 2024-11-05 at 12 28 32 PM" src="https://github.com/user-attachments/assets/2f3584c5-9ce1-4ec4-b8a2-55c2f71c99de">

[Supplementaries](https://github.com/MehVahdJukaar/Supplementaries)

<img width="522" alt="Screenshot 2024-11-05 at 12 32 16 PM" src="https://github.com/user-attachments/assets/69c55758-d03c-4ff9-986f-e56cb29d46aa">

[Amethyst Imbuement](https://github.com/fzzyhmstrs/ai)

<img width="697" alt="Screenshot 2024-11-05 at 12 37 25 PM" src="https://github.com/user-attachments/assets/0d14dd75-49b5-4cf7-9710-aac79d8f9291">

Axiom (could not find source code)

<img width="482" alt="Image" src="https://github.com/user-attachments/assets/0c70bb03-679f-4e20-83e6-1c5e3fbc8aeb" />

[Sidekick](https://git.sleeping.town/unascribed-mods/Sidekick)

![Image](https://github.com/user-attachments/assets/caae98b0-2e79-4bc9-a2d8-8807f92da3c2)

I'll reply to this issue if I find more mods that could have added compatibility

**Comment by blryface** [comment]
###### *edit: Add github links to all mods and fix some issues with the description*

**Comment by NormalFery** [comment]
I totally haven't been putting off this comment because I kept forgetting until now but both Mod Menu and Inventory Profiles Next have compat already done (Since iirc 2.0.0 i think would bee in modern versioning / 18.1.0 )? , it just hasn't been added to Legacy (1.20.1), as for the rest of the mods, it'll be added after #11 is fixed entirely since fixing / making assets for the base game is more important than mod compat imo

aside from that yeah these mods should have compat and I still regret not having done it before release

**Comment by blryface** [comment]
Can't wait for 3.0 then!

**Comment by blryface** [comment]
###### *edit: checklist*

**Comment by NormalFery** [comment]
As of 3bc934c , Origins (Fabirc, forge should work out of the box unless somehow it's literally completely different), Enchancement, and Animal Feeding Trough have been completed 👍

**Comment by blryface** [comment]
###### *edit: update checklist*

**Comment by Ampflower** [comment]
I'ma say that Sidekick could be done too.

I feel like this issue would be best split up into multiple btw.

**Comment by Thinkseal** [comment]
> I'ma say that Sidekick could be done too.

added

> I feel like this issue would be best split up into multiple btw.

I feel like that would create a lot of separate issues that would be harder to go through maybe a potential solution for making this one less long is to remove the images for the mods that have been implemented?

**Comment by Samalando** [comment]
all malilib based mods are compatible, except custom buttons

![Image](https://github.com/user-attachments/assets/8a2032f2-2a6d-4166-aa2a-02b6906eb49a)
![Image](https://github.com/user-attachments/assets/5a7a90a9-e3bb-4b87-9aa6-31a98b05e9de)

**Comment by Ampflower** [comment]
> > I feel like this issue would be best split up into multiple btw.
> 
> I feel like that would create a lot of separate issues that would be harder to go through maybe a potential solution for making this one less long is to remove the images for the mods that have been implemented?

I wouldn't necessarily say that's a problem; if anything it can be more helpful to just track by each individual mod especially if a mod has a lot of textures.

**Comment by Thinkseal** [comment]
> I wouldn't necessarily say that's a problem; if anything it can be more helpful to just track by each individual mod especially if a mod has a lot of textures.

Just to make sure blurry is fine with it and I can split up this into multiple issues

**Comment by Ampflower** [comment]
Probably more so @NormalFery's call there, their repo.

**Comment by blryface** [comment]
Not really sure why **I** would need to give permission rather than Ferh

**Comment by Thinkseal** [comment]
> Not really sure why **I** would need to give permission rather than Ferh

I am tired right now and not thinking that much

**Comment by NormalFery** [comment]
> > I feel like this issue would be best split up into multiple btw.
> 
> I feel like that would create a lot of separate issues that would be harder to go through maybe a potential solution for making this one less long is to remove the images for the mods that have been implemented?

I feel like multiple issues would be the best way to go imo, there are a few mods (Cough cough Create) that have like a THOUSAND different GUIs that can't be cleanly tracked in a single issue, I'm all in for splitting this into separate issues for mods that haven't been completed.

It'd also allow me to add x or y mods that need the GUI changes as well (Like AE2, Spectrum, or similar) more easily and also track progress of those, so imo, yes, all in on splitting.

**Comment by Ampflower** [comment]
That sounds fair to me.

**Comment by Thinkseal** [comment]
Before I split this into a ton of issues, can you double check that all of the ones that have not been completed actually have not been completed so I don't make an issue for one that has been completed

**Comment by NormalFery** [comment]
> Before I split this into a ton of issues, can you double check that all of the ones that have not been completed actually have not been completed so I don't make an issue for one that has been completed

As of 96868cd (Latest commit in Legacy 1.20.1 , where mod compat is targeted to), the current completed ones in this issue should be accurrate, so you can do the new issues

**Comment by NormalFery** [comment]
Given how now there's multiple issues to track this, I'll close this issue later on as completed, and will comment on each separate issue depending on status on updates.

**Comment by Thinkseal** [comment]
replaced with #14, #15, #16, #17, #18, #19, and #20

**Comment by blryface** [comment]
it shouldn't be closed

> I'll close this issue later on as completed, and will comment on each separate issue depending on status on updates.

**Comment by NormalFery** [comment]
> it shouldn't be closed

Why shouldn't it? As far as I'm aware, this has been replaced with new issues, and it will no longer be used for anything, as far as I know.

**Comment by blryface** [comment]
hm I think I misunderstood what you said

**Comment by blryface** [comment]
I thought you meant you'd keep this one open until all the other issues were solved

**Comment by blryface** [comment]
I don't think it should be closed as *completed* tho?
